aspnetcore/mvc/controllers/areas.md

```diff
@@ -185,13 +185,12 @@ Browsing to *http://<yourApp>/products*, the `Index` action method of the `HomeC
 
 ## Publishing Areas
 
-To publish all views of the areas folder, in the `project.json` file include an entry in the `publishOptions`'s `include` node like below:
+To publish all views of the areas folder, in the *.csproj* file include `Areas\**\*.cshtml` n the following markup:
 
-```text
-"publishOptions": {
-     "include": [
-       "Areas/**/*.cshtml",
-       ....
-       ....
-     ]
+```xml
+  <ItemGroup>
+    <Content Update="wwwroot\**\*;**\*.cshtml;Areas\**\*.cshtml;">
```

Comment:
Yes, that's right. It is already included in websdk.

https://github.com/aspnet/websdk/blob/dev/src/Web/Microsoft.NET.Sdk.Web.ProjectSystem.Targets/netstandard1.0/Microsoft.NET.Sdk.Web.ProjectSystem.props#L25-L30